### PR TITLE
Add new datasets in Dec. 2021

### PR DESCRIPTION
**Pull request by PaulWessel**
This updates the _earth_relief_ data, adds the related _earth_synbath_, and then add new data sets for Earth (faa, geoid, vgg, mag, mag4km, and wdmam). Note: I have not tried to build the docs.

### CODE REVIEW

**Comment by meghanrjones** [comment]
Docs look good.

We should hold off on merging until they are placed on the server. Just to confirm - the terms allow all these to be re-distributed?

**Comment by PaulWessel** [comment]
Yes, I checked with both mag groups and got their preferred CPTs and David is onboard of course.
The Gebco grid is placed in the public domain so we do not need to request any approval but we may want to get their preferred CPT if any.  I started downloading their grid last night but timed out...

**Comment by PaulWessel** [comment]
Great @Esteban82, can you make a PR for dcw-gmt? That way we can help with fixing these and other spelling issues form the shape file.

**Comment by PaulWessel** [comment]
Hi @meghanrjones might you be able to build the updated branch so we can have a look at what might be missing if anything.  I think I have added all the data set descriptions and images.

**Comment by meghanrjones** [comment]
> Hi @meghanrjones might you be able to build the updated branch so we can have a look at what might be missing if anything. I think I have added all the data set descriptions and images.

Yes, I'll try to get Vercel set up for previews.

File: docs/index.rst
Comment (meghanrjones) [inline docs/index.rst:9]:
```suggestion
    earth-gebco.rst
    earth-geoid.rst
```

**Comment by PaulWessel** [comment]
@meghanrjones, might you be able to edit earth-relief.rst and replace the XXX with a valid link to the auto-resolution place in the GMT docs? I can then use that in other places and finish up those RST files.

File: docs/earth-gebco.rst
Comment (PaulWessel) [reply docs/earth-gebco.rst:48]: Yes, same SRTM tiles. When we use the new auto-resolution syntax @earth_gebco then the resolution depends on -R -J etc and I dont think we want to be in a situation where we go from 15s to 3s and the script will complain there is no such resolution.  Note: There are no exatra files. GMT will replace earth_gebco with eart_relief for 3s and1s.

File: docs/earth-faa.rst
Comment (PaulWessel) [reply docs/earth-faa.rst]: Yes, that is a bit silly.  I just changed all to not have fig in them.

**Review by meghanrjones — APPROVED** [review]

**Pull request merged**
